styles/globals.css

```
@@ -0,0 +1,95 @@
+    --foreground: black;
+    --hoverbg: #f3f4f6;
+    --dvh: 100dvh;
+    --rootBg: #f4f4f5;
+    --borderColor: #cecece;
+    --buttonBg: #e0e0e0;
+    overflow: hidden;
+    user-select: none;
+    -ms-user-select: none;
+    -webkit-user-select: none;
+    -webkit-user-drag: none;
+    -khtml-user-drag: none;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)